components/button/ButtonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import clsx from 'clsx'
+import { Link } from 'expo-router'
+
+interface Props {
+  content: string
+  url: string
+  isAbsolute?: boolean
+}
+
+const ButtonLink = ({ content, url, isAbsolute }: Props) => {
```

Comment:
evitar usar const para la declaracion de un componente, en su lugar : 
```suggestion
function ButtonLink({ content, url, isAbsolute }: Props) {
```